jabgui/src/main/java/org/jabref/gui/preferences/general/GeneralTabViewModel.java

```diff
@@ -161,13 +163,32 @@ public GeneralTabViewModel(DialogService dialogService, GuiPreferences preferenc
                         Localization.lang("Remote operation"),
                         Localization.lang("You must enter an integer value in the interval 1025-65535"))));
 
+        enableHttpPortValidator = new FunctionBasedValidator<>(
+                enableHttpPortProperty,
+                input -> {
+                    try {
+                        int portNumber = Integer.parseInt(enableHttpPortProperty().getValue());
+                        return RemoteUtil.isUserPort(portNumber);
+                    } catch (NumberFormatException ex) {
+                        return false;
+                    }
+                },
+                ValidationMessage.error("%s > %s %n %n %s".formatted(
+                        Localization.lang("Network"),
+                        Localization.lang("Remote operation"),
+                        Localization.lang("You must enter an integer value in the interval 1025-65535"))));
```

Comment:
I think, it can be `Localization.lang("You must enter an integer value in the interval 1025-65535")` only, no need for network and remote operation.

Maybe, you can even merge it with remotePortValidator  to avoid code duplicatoin.